estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,80 @@
+from odoo import api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class EstatePropertyOffer(models.Model):
+    _name = 'estate.property.offer'
+    _description = 'Estate Property Offer'
+    _order = 'price desc'
+
+    price = fields.Float()
+    status = fields.Selection(
+        string="Status",
+        selection=[("accepted", "Accepted"), ("refused", "Refused")],
+        copy=False,
+    )
+    validity = fields.Integer(default=7, string="Validity (days)")
+    date_deadline = fields.Date(
+        compute="_compute_deadline", inverse="_inverse_deadline", string="Deadline"
+    )
+
+    # Relations
+    partner_id = fields.Many2one("res.partner", string="Partner")
+    property_id = fields.Many2one("estate.property", string="Property")
+    property_type_id = fields.Many2one(
+        related="property_id.property_type_id", store=True
+    )
+
+    # Methods
+    @api.depends("validity")
+    def _compute_deadline(self):
+        for offer in self:
+            if offer.create_date:
+                offer.date_deadline = fields.Date.add(
+                    offer.create_date, days=offer.validity
```

Comment:
To be changed in some other spots in the code
```suggestion
                    offer.create_date, days=offer.validity,
```